sources/api/netdog/src/networkd/devices/mod.rs

```diff
@@ -0,0 +1,15 @@
+//! The device module contains the structures representing the latest version of configuration for
+//! interfaces, bonds, and VLANs.
+mod bond;
+mod interface;
+mod vlan;
+
+pub(crate) use bond::NetworkDBond;
+pub(crate) use interface::NetworkDInterface;
+pub(crate) use vlan::NetworkDVlan;
+
+pub(crate) enum NetworkDDevice {
+    Interface(NetworkDInterface),
+    BondDevice(NetworkDBond),
```

Comment:
Adding a comment here to remind myself to remove the `Device` from `Bond` and `Vlan`... :facepalm: